docs/topic_guides/customising_comparisons.ipynb

```diff
@@ -213,7 +213,90 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Method 2: `ComparisonLevels`\n",
+    "## Method 2: Using the `ComparisonTemplateLibrary`\n",
+    "\n",
+    "The `ComparisonTemplateLibrary` is very similar to `ComparisonLibrary` in that it contains pre-baked similarity functions for each backend (DuckDB, Spark, etc.) to cover common use cases.\n",
```

Comment:
I think this addition and subsequent renumbering will break a few links: any link previously with `#method-3-providing-the-spec...` and `#method-2-comparisonlevels` will need to have their numbers incremented

Reply:
I thought I had caught all of them, but missed one. Pushed now

Reply:
Nice one, thanks!